### DISCUSSION TITLE
Are there any known issues with sending key events to the game?

### DISCUSSION THREAD

**Discussion by ScottBouloutian** [tanjeffreyz/auto-maple#27] [Q&A] [answered]
Hi, I started researching ways machine learning could be applied to maplestory and came across this project. I am not able to get auto-maple to send key events to the game, are there any known issues with this? I wrote a simple routine to test auto-maple, if I open notepad I see it typing the "e" key trying to teleport my Kanna, however if I focus back on the game window nothing happens. I am on Windows 11, I think I am going to debug further in the code that sends key presses.

**Comment by whytrade** [comment, marked as answer]
Nothing wrong with the code that sends key presses, you need to have auto-maple open with elevated privileges

**Comment by noobishh** [comment]
hi was wondering if youre still using the macro? thanks!
